lib/tasks/projects.rake

```diff
@@ -218,4 +218,30 @@ namespace :projects do
       REDIS.set("go:update:latest_updated_id", projects.last.id)
     end
   end
+
+  desc "Verify Pypi Projects"
+  task :verify_pypi_projects, [:count] => :environment do |_task, args|
+    args.with_defaults(count: 50)
+
+    project_ids_and_names = Project
+      .select("min(id)", "lower(name)")
+      .where(platform: "Pypi")
+      .group("lower(name)")
+      .having("count(name) > 1")
+      .order("min(id)") # order this query so we can run through chunks of the results if needed
+      .limit(args[:count])
+      .pluck("min(id)", "lower(name)")
```

Comment:
Yeah I can think through how to handle hyphens and underscore since those are also treated as equivalent.

From https://peps.python.org/pep-0426/#name

> All comparisons of distribution names MUST be case insensitive, and MUST consider hyphens and underscores to be equivalent.

Reply:
Updated the query to also handle names where hyphens and underscores can be treated as the same in c808e9ecc4aa85a2d0666c094062561cf1f5e4f6